From 6fcd53f1234a0871151033db54a0b231c0898dcc Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Tue, 10 Oct 2023 10:21:31 +0800
Subject: [PATCH 06/14] SM1&G12B: cold start time optimization. [2/3]

PD#SWPL-139535

Problem:
Cold start time optimization.

Solution:
1. Improve CPU performance during startup.

Verify:
local PB

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: I874394ba96134e9be9ae585eb84109f1e5bc1af8
---
 recipes-kernel/linux/5.15/g12b.cfg | 1 +
 recipes-kernel/linux/5.15/sm1.cfg  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/recipes-kernel/linux/5.15/g12b.cfg b/recipes-kernel/linux/5.15/g12b.cfg
index 7dda5276..e120cc72 100644
--- a/recipes-kernel/linux/5.15/g12b.cfg
+++ b/recipes-kernel/linux/5.15/g12b.cfg
@@ -30,6 +30,7 @@ CONFIG_AMLOGIC_SND_SOC_TAS5805=m
 CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y
 CONFIG_AS_HAS_NON_CONST_LEB128=y
 CONFIG_AMLOGIC_MEDIA_CAMERA=m
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
 
 #Remove some configs
 # CONFIG_AMLOGIC_PINCTRL_MESON_S4 is not set
diff --git a/recipes-kernel/linux/5.15/sm1.cfg b/recipes-kernel/linux/5.15/sm1.cfg
index 26ea121c..b617dc3e 100644
--- a/recipes-kernel/linux/5.15/sm1.cfg
+++ b/recipes-kernel/linux/5.15/sm1.cfg
@@ -12,6 +12,7 @@ CONFIG_BT_RFCOMM=y
 # CONFIG_CC_HAVE_STACKPROTECTOR_SYSREG is not set
 # CONFIG_BT_HCIBTSDIO is not set
 CONFIG_EXFAT_FS=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
 CONFIG_ARM64_ERRATUM_2441007=y
 CONFIG_REGULATOR_GPIO=m
 CONFIG_AMLOGIC_DRM_VPU=y
-- 
2.25.1

